packages/picasso/src/TagSelector/story/index.jsx

```diff
@@ -23,6 +23,10 @@ page
     'TagSelector/story/InitialSetValue.example.tsx',
     'Initially set value'
   )
+  .addExample('TagSelector/story/KeepOpen.example.tsx', {
+    title: 'Keep options open',
+    takeScreenshot: false,
+  }) // picasso-skip-visuals
```

Comment:
```suggestion
  })
```

this comment is not needed anymore, we use the `takeScreenshot: false` above